src/core_plugins/kibana/public/doc/index.html

```diff
@@ -1,47 +1,75 @@
 <div ng-controller="doc" class="app-container">
+  <div class="kuiLocalNav">
```

Comment:
Hm, you probably have a broader vision for the visual identity of Kibana than me. Subjectively, the whole application just feels more solid and cohesive the more consistent the frame around all the pages is. My intention was to propose something like the below restructuring as a resolution to #10198 next:

![image](https://cloud.githubusercontent.com/assets/973741/22694626/3d9f11c6-ed48-11e6-99de-7c89ad470829.png)